atmoz-sftp.yaml

```diff
@@ -0,0 +1,82 @@
+#nolint:valid-pipeline-git-checkout-commit,valid-pipeline-git-checkout-tag
+package:
+  name: atmoz-sftp
+  version: 0.0.0_git20250226
+  epoch: 0
+  description: Various scripts from https://github.com/atmoz/sftp to help run SFTP in a container
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - bash
+      - openssh-client # Includes `sftp` command
+      - openssh-keygen # used in entrypoint script
+      - openssh-pam
+      - openssh-sftp-server
+      - posix-libc-utils # For `getent` command
+      - shadow
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libedit-dev
+      - linux-pam-dev
```

Comment:
I was looking through the upstream repo and I"m not sure why it would even need pam at all. None of the particulars in the sshd config use it and the user creation is all happening against shadow. We could try doing this w/o the ssh and pam changes.